dnf/cli/commands/autoremove.py

```diff
@@ -37,7 +37,7 @@ class AutoremoveCommand(commands.Command):
                    'autoremove-na': hawkey.FORM_NA,
                    'autoremove-nevra': hawkey.FORM_NEVRA}
 
-    aliases = ('autoremove',) + tuple(nevra_forms.keys())
+    aliases = ('autoremove', 'auto-rm',) + tuple(nevra_forms.keys())
```

Comment:
I suggest taking the dash out. We don't have one in `autoremove`, and `autorm` is still unique.